examples/resources/service_dynamic_snippet_content_manage_snippets.tf

```diff
@@ -0,0 +1,15 @@
+#...
+
+resource "fastly_service_dynamic_snippet_content" "my_dyn_content" {
+  for_each = {
+    for d in fastly_service_vcl.myservice.dynamicsnippet : d.name => d if d.name == "My Dynamic Snippet"
+  }
+  service_id       = fastly_service_vcl.myservice.id
+  snippet_id       = each.value.snippet_id
+  manage_snippets = true
+  content          = "if ( req.url ) {\n set req.http.my-snippet-test-header = \"true\";\n}"
+
+  lifecycle {
```

Comment:
As mentioned above I don't know if `ignore_changes` is expected to work when the new `manifest_*` is in place?